src/main/java/com/backbase/ct/bbfuel/setup/CapabilitiesDataSetup.java

```diff
@@ -40,6 +43,7 @@
 
     private final UserContextService userContextService;
     private final UserContextPresentationRestClient userContextPresentationRestClient;
+    private final UserPresentationRestClient userPresentationRestClient;
```

Comment:
The `userPresentationRestClient` and `accountsIntegrationRestClient` are no longer needed in this class.